src/proxy-ui-api/src/main/java/org/niis/xroad/restapi/service/AccessRightService.java

```diff
@@ -222,7 +222,7 @@ public void deleteSoapServiceAccessRights(ClientId clientId, String fullServiceC
      */
     private List<AccessRightHolderDto> addSoapServiceAccessRights(ClientId clientId, String fullServiceCode,
```

Comment:
Lets add the missing @throws LocalGroupNotFoundException to javadoc.

Reply:
Also DuplicateAccessRightException is missing too. Perhaps add that too.

Reply:
Added both of them